src/constants/sounds.ts

```diff
@@ -3,6 +3,11 @@ export type MusicTrack = {
   url: string;
 };
 
+export const LOSE_SFX = 11;
```

Comment:
Check comment in Scores component.
instead of storing LOSE_SFX and WIN_SFX  as constants you can store the index, and size of the sub array that you want to get in order to select a random sound. Just leave a comment explaining the relation so that if we change the array of TrackIDs, the constant also gets updated.